prowler/providers/aws/services/securityhub/securityhub_service.py

```diff
@@ -85,6 +87,18 @@ def _describe_hub(self, regional_client):
                 f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
 
+    def _list_tags(self, resource: any):
```

Comment:
Please, check here not to do it for Hub with `NOT_AVAILABLE` status.

Reply:
Since the ARN would be an account ARN and therefore, not valid.

Reply:
Solved!